nio-core/src/main/scala/zio/nio/core/charset/CharsetDecoder.scala

```diff
@@ -132,10 +132,13 @@ final class CharsetDecoder private (val javaDecoder: j.CharsetDecoder) extends A
               } yield decodedChars ++ remainderChars
 
             inPull.foldM(
-              _.map(e => ZIO.fail(Some(e))).getOrElse {
+              _.map(e => ZIO.fail(Option(e))).getOrElse {
```

Comment:
`Some` looks more appropriate.

Reply:
It complained about `Some`, it required `Option` which is why I changed it.

Reply:
:+1: